src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -80,6 +81,10 @@ static const NLTypeSystem empty_type_system = {
         .types = empty_types,
 };
 
+static const NLType rtnl_link_info_data_batadv_types[] = {
+        [IFLA_BATADV_ALGO_NAME] = { .type = NETLINK_TYPE_STRING, .size = 20 },
```

Comment:
FWIW: This parameter is still being discussed by the batman-adv developers and has not been merged into batman-adv yet.

Reply:
Merged upstream in https://git.open-mesh.org/batman-adv.git/commit/e5f7ebc0d8a76b9aab3dd7f80784c233169d8069 and will be part of batman-adv 2021.0.